<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calculadora</class>
 <widget class="QMainWindow" name="calculadora">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculadora</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>149</x>
      <y>59</y>
      <width>311</width>
      <height>231</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_num1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Número 1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_num1">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_num2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Número 2</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_num2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_calcular">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>170</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Calcular</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_operacao">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Operação</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_operacao">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>120</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
